src/pages/Home.js

```diff
@@ -44,13 +66,20 @@ export default function Home(props) {
 
   return (
     <main>
-      <h1>Welcome screen</h1>
-      <h2>Add a new list:</h2>
-      <button onClick={handleClick}>Add List</button>
-      <h2>Join an existing list</h2>
+      <h1>Welcome To CLEVER NAME TO BE DETERMINED List!</h1>
+      <p>
+        CNTBD is here to add a little more organization to your grocery list. Or
+        lists!
+      </p>
+      <p>To start, either:</p>
+      <h2>Create a brand new list</h2>
```

Comment:
This is more of a question for @yenly and mentors: is it semantically appropriate to have an `<p>` tags sandwiched in between an `<h1>` and `<h2>` ? I know that you have to go in order from `<h1>`, `<h2>`, `<h3>` etc. But just curious if it's okay to have other elements between them?

Reply:
Thanks Yenly!